build.sh

```diff
@@ -17,10 +17,16 @@
 # under the License.
 
 ##############################################################
-# This script is used to compile Apache Doris(incubating)
-# Usage:
-#    sh build.sh        build both Backend and Frontend.
-#    sh build.sh -clean clean previous output and build.
+# This script is used to compile Apache Doris(incubating).
+# Usage: 
+#    sh build.sh --help
+# Eg:
+#    sh build.sh                        build all
+#    sh build.sh  --be                  build Backend without clean
+#    sh build.sh  --fe --clean          clean and build Frontend and Spark Dpp application
+#    sh build.sh  --fe --be --clean     clean and build Frontend, Spark Dpp application and Backend
+#    sh build.sh  --spark-dpp           build Spark DPP application alone
+#    sh build.sh  --fe --ui             build Frontend web ui with npm
```

Comment:
Please add a new entry:
# sh build.sh  --fe --be --ui --clean     clean and build Frontend, Spark Dpp application, Backend and web UI